content/administration/install/install.rst

```diff
@@ -785,6 +789,10 @@ A typical way to run the server would be:
 Where `CommunityPath` is the path of the Odoo Community installation
 and `mydb` is the default database to serve on `localhost:8069`.
 
+.. warning:: Some versions of python on Mac might lead to the following error message:
+          `current limit exceeds maximum limit`. In that case, add the following
+          parameter to avoid the problem: `--limit-memory-hard 0`.
```

Comment:
```suggestion
.. warning::
   Some versions of python on Mac might lead to the following error message: `current limit exceeds
   maximum limit`. In that case, add the following parameter `--limit-memory-hard 0` to avoid the
   problem.
```